pkg/azure/acr.go

```diff
@@ -4,11 +4,31 @@ import (
 	"context"
 	"errors"
 	"fmt"
-
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/authorization/armauthorization/v2"
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerregistry/armcontainerregistry"
 	"github.com/azure/spin-aks-plugin/pkg/logger"
 )
 
+type Role struct {
+	Name string
+	ID   string
+}
+
+const (
+	// tbarnes94: placeholder for now
+	subscriptionId = "8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8"
```

Comment:
We shouldn't be commiting subscription ids at all? Should probably scrub this from commit history too

Reply:
agree - will try squashing commits